app/components/view_component/form/base_component.rb

```diff
@@ -7,6 +7,8 @@ class << self
         attr_accessor :default_options
       end
 
+      include ClassNamesHelper if Gem::Version.new(Rails::VERSION::STRING) < Gem::Version.new("6.1")
```

Comment:
```suggestion
      if Gem::Version.new(Rails::VERSION::STRING) < Gem::Version.new("6.1")
        require "view_component/form/class_names_helper"
        include ClassNamesHelper
      end
```

I suggest to only require helper on Rails < 6.1

Reply:
Also maybe we could uncomment `# opts[:class] = class_names(options[:class], html_class) if (html_class || options[:class]).present?` ?